docs/guides/javascript.md

```diff
@@ -55,14 +55,19 @@ Spectral comes with some rules and functions that are very specific to OpenAPI v
 ```js
 const { Spectral } = require('@stoplight/spectral');
 const { parseWithPointers } = require("@stoplight/yaml");
-const { oas3Functions, rules: oas3Rules } = require('@stoplight/spectral/dist/rulesets/oas3');
+const { oas3Functions } = require('@stoplight/spectral/dist/rulesets/oas3');
```

Comment:
We're going ahead with this feature first, then getting rid of functions in #430 afterwards? I thought I saw you mention doing it the other way around but it does not matter to me. Just making sure that is the plan.

Reply:
Yeah, that's the plan.